docs/pages/guides/authentication.mdx

```diff
@@ -127,6 +128,115 @@ export default function App() {
 </Box>
 {/* End IdentityServer 4 */}
 
+<Box
+  name="Asgardeo"
+  createUrl="https://wso2.com/asgardeo/docs/guides/#developer-guide"
+  image={ASSETS.asgardeo}
+>
+
+| Website                       | Provider | PKCE      | Auto Discovery |
+| ----------------------------- | -------- | --------- | -------------- |
+| [Get Your Config][c-asgardeo] | OpenID   | Supported | Available      |
+
+[c-asgardeo]: https://console.asgardeo.io/
+
+- Make sure to check `Public Client` in the console.
+- Check `Implicit` in the Allowed grant types section.
+
+<Tabs tabs={["Auth Code"]}>
+
+<Tab>
+
+{/* prettier-ignore */}
+```tsx
+import { useState, useEffect } from 'react';
+import { StyleSheet, Text, View, Button, Alert } from 'react-native';
+import * as AuthSession from "expo-auth-session";
+import * as WebBrowser from "expo-web-browser";
+import jwtDecode from "jwt-decode";
+
+WebBrowser.maybeCompleteAuthSession();
+
+const useProxy = true;
+const redirectUri = AuthSession.makeRedirectUri({ useProxy });
+
+/* @info <strong>Client ID:</strong> This can be found on the protocol tab of Asgardeo Console */
+const CLIENT_ID = "YOUR_CLIENT_ID";
+/* @end */
+
+export default function App() {
+
+  /* @info <strong>Auto Discovery URL:</strong> This can be found on the info tab of Asgardeo Console. Copy the link under `Issuer` */
+  const discovery = AuthSession.useAutoDiscovery('https://api.asgardeo.io/t/<YOUR_ORG_NAME>/oauth2/token');
+  /* @end */
+  const [decodedToken, setDecodedToken] = useState({});
+
+  const [request, result, promptAsync] = AuthSession.useAuthRequest(
+        {
+            redirectUri,
+            clientId: CLIENT_ID,
+            responseType: "token",
+            scopes: ["openid", "profile", "email", "offline_access"]
```

Comment:
I don't think we have `offline_access` scope in Asgardeo

Reply:
Added that in case we need it in the future implementations. I'll remove it for now